frontend/src/components/Action/ActionNotesList.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import { Box } from '@material-ui/core'
+
+const notes = [
+    {
+        id: 'hfjkhfkf78f6s9fhj',
+        writerName: 'Anna Therese Asheim',
+        message: 'Kan du se på dette Andreas Rønning?',
+        date: '2021-01-10T11:27:08.024+01:00',
+    },
+]
```

Comment:
I think this should be place higher in the react hierarchy. This component could actually be finished in this PR. 

```suggestion
const notes: Note[] = [
    {
        id: 'hfjkhfkf78f6s9fhj',
        writerName: 'Anna Therese Asheim',
        message: 'Kan du se på dette Andreas Rønning?',
        date: '2021-01-10T11:27:08.024+01:00',
    },
]
```

Reply:
I can make the changes but I assume I have to tweak it when connecting to the db since I don't know the real property names.

Reply:
What do you mean by higher in the react hierarchy? In the file tree?

Reply:
Yes, in the file tree. You can find the real property names in `frontend/src/api/models.ts`.

Reply:
I started on this but it requires some more work than just selecting two different property names (because of the Notes type). It's not a big job, but I might as well do it when I start digging into the Notes-universe in the next task later today.